measure_extinction/utils/make_obsdata_from_model.py

```diff
@@ -482,14 +521,26 @@ def make_obsdata_from_model(
                 "g-",
             )
 
+            (indxs,) = np.where(rb_iue["NPTS"] > 0)
+            ax.plot(rb_iue["WAVELENGTH"][indxs].to(u.micron), rb_iue["FLUX"][indxs], "r-")
+
             (indxs,) = np.where(rb_nrc["NPTS"] > 0)
             ax.plot(rb_nrc["WAVELENGTH"][indxs].to(u.micron), rb_nrc["FLUX"][indxs], "c-")
 
             (indxs,) = np.where(rb_lrs["NPTS"] > 0)
             ax.plot(rb_lrs["WAVELENGTH"][indxs].to(u.micron), rb_lrs["FLUX"][indxs], "r:")
 
+            (indxs,) = np.where(rb_niriss["NPTS"] > 0)
+            ax.plot(rb_niriss["WAVELENGTH"][indxs].to(u.micron), rb_niriss["FLUX"][indxs], "r-")
+
+            (indxs,) = np.where(rb_nrc["NPTS"] > 0)
+            ax.plot(rb_nrc["WAVELENGTH"][indxs].to(u.micron), rb_nrc["FLUX"][indxs], "r-")
+
+            (indxs,) = np.where(rb_miri_lrs["NPTS"] > 0)
+            ax.plot(rb_miri_lrs["WAVELENGTH"][indxs].to(u.micron), rb_miri_lrs["FLUX"][indxs], "r--")
+
             (indxs,) = np.where(rb_mrs["NPTS"] > 0)
-            ax.plot(rb_mrs["WAVELENGTH"][indxs].to(u.micron), rb_mrs["FLUX"][indxs], "r-")
+            ax.plot(rb_mrs["WAVELENGTH"][indxs].to(u.micron), rb_mrs["FLUX"][indxs], "g-")
```

Comment:
Keep in mind that plotting in this way will connect the good regions with each other. I.e., where NPTS=0, you will not see gaps in the plots, but "fake" data. I just want to make sure whether this is how you intended this plotting (here and all above).

Reply:
Yep.  Shouldn't be any NPTS=0 given this is model data.